src/test/java/edu/kis/powp/jobs2d/commands/SubscribeCommandsCounterVisitor.java

```diff
@@ -0,0 +1,27 @@
+package edu.kis.powp.jobs2d.commands;
+
+import edu.kis.powp.jobs2d.command.CommandsCounterVisitor;
+import edu.kis.powp.jobs2d.command.manager.DriverCommandManager;
+import edu.kis.powp.jobs2d.features.CommandsFeature;
+import edu.kis.powp.observer.Subscriber;
+
+import java.util.logging.Logger;
+
+public class SubscribeCommandsCounterVisitor implements Subscriber {
+	DriverCommandManager manager = CommandsFeature.getDriverCommandManager();
```

Comment:
should be private

dependency with CommandsFeature is not necessary (could be passed in construtor)